web/client/src/components/form/TokenDisplay.jsx

```diff
@@ -12,60 +12,74 @@ import PropTypes from "prop-types";
 import "../../styles/form.css";
 import { Alert } from "@material-ui/lab";
 import CloseIcon from "@material-ui/icons/Close";
+
 /**
- *
  * @param {string} props token being passed for display
  * @return {Grid} returns a Grid component that contains the page
  */
 export default function TokenDisplay(props) {
-  const { token, responseVisable } = props;
```

Comment:
I think this format of unwrapping is preferred. I could be wrong tho

Reply:
If there's no preference, we should at least keep it the above way for consistency

Reply:
Yea that makes sense, I changed it back